src/tests/user.test.js

```diff
@@ -191,4 +194,81 @@ describe('User Signin', () => {
             console.log(err);
           })
     });
-})
\ No newline at end of file
+})
+
+
+describe('/GET users', () => {
+
+	it('should return all users', () => {
+    
+    const user_credentials = {
+      "email": "keza@gmail.com",
+      "password": "1234567avb$#8"
+    }
+    chai.request(app).post('/user/signin').send(user_credentials).then((login_response) => {
+          token = 'Bearer ' + login_response.body.token;
+          chai.request(app).get('/user/').set('Authorization', token)
+              .end((err, res) => {
+                expect(res).to.have.status(200);
+              })
+          }).catch(err => {
+              console.log(err);
+          })
+  });
+})
+
+describe('/GET/:id user', () => {
+  it('it should GET a user by the given id', () => {
+      let user = new User({firstname: "kalisa", lastname: "wilson", email: "wizo@gmail.com", password: "12345678#$"});
+      user.save((err, user) => {
+          chai.request(app)
+        .get('/user/' + user.id)
+        .send(user)
+        .set('Authorization', token)
```

Comment:
You have not generated a token for this test request, please generate one token to be used throughout the test

Reply:
Also, enforce styling in your tests, do not write everything on one line